tasks/holesky/001-pectra-defense/NestedSignFromJson.s.sol

```diff
@@ -0,0 +1,114 @@
+import {Simulation} from "@base-contracts/script/universal/Simulation.sol";
+import {NestedSignFromJson as OriginalNestedSignFromJson} from "script/NestedSignFromJson.s.sol";
+import {CouncilFoundationNestedSign} from "script/verification/CouncilFoundationNestedSign.s.sol";
+
+// Monorepo deps
+import {IFaultDisputeGame} from "@eth-optimism-bedrock/interfaces/dispute/IFaultDisputeGame.sol";
+import {IPermissionedDisputeGame} from "@eth-optimism-bedrock/interfaces/dispute/IPermissionedDisputeGame.sol";
+import {IDisputeGameFactory} from "@eth-optimism-bedrock/interfaces/dispute/IDisputeGameFactory.sol";
+// import {IProxyAdmin} from "@eth-optimism-bedrock/interfaces/universal/IProxyAdmin.sol";
+import {GameTypes, GameType} from "@eth-optimism-bedrock/src/dispute/lib/Types.sol";
+// import {ISystemConfig} from "@eth-optimism-bedrock/interfaces/L1/ISystemConfig.sol";
+import {StandardValidatorV180, IProxyAdmin, ISystemConfig} from "@eth-optimism-bedrock/src/L1/StandardValidator.sol";
+
+contract NestedSignFromJson is OriginalNestedSignFromJson, CouncilFoundationNestedSign {
+
+    IDisputeGameFactory constant OP_DGF = IDisputeGameFactory(0xF1408Ef0c263F8c42CefCc59146f90890615A191);
+    ISystemConfig constant SYS_CFG = ISystemConfig(0x9FB5e819Fed7169a8Ff03F7fA84Ee29B876D61B4);
+    IProxyAdmin constant PROXY_ADMIN_ADDRESS = IProxyAdmin(0xbD71120fC716a431AEaB81078ce85ccc74496552);
+    IProxyAdmin constant SUPERCHAIN_PROXY_ADMIN = IProxyAdmin(0xFeE222a4FA606A9dD0B05CD0a8E1E40e60FD809a);
+
+    mapping(IDisputeGameFactory => mapping(GameType => IFaultDisputeGame.GameConstructorParams)) public beforeParams;
+
+    function setUp() public {
+        addAllowedStorageAccess(address(OP_DGF));
+        beforeParams[OP_DGF][GameTypes.PERMISSIONED_CANNON] =
+            getGameConstructorParams(IFaultDisputeGame(address(OP_DGF.gameImpls(GameTypes.PERMISSIONED_CANNON))));
+
+        beforeParams[OP_DGF][GameTypes.CANNON] =
+            getGameConstructorParams(IFaultDisputeGame(address(OP_DGF.gameImpls(GameTypes.CANNON))));
+    }
+
+    function _postCheck(Vm.AccountAccess[] memory accesses, Simulation.Payload memory) internal view override {
+        console.log("Running post-deploy assertions");
+
+        // Does not work on Holesky because the addresses are not in the registry
+        // accesses.decodeAndPrint();
+
+        checkStateDiff(accesses);
+
+        // get the before and after game params for the permissioned game
+        IFaultDisputeGame.GameConstructorParams memory beforeParams_ =
+            beforeParams[OP_DGF][GameTypes.PERMISSIONED_CANNON];
+        IFaultDisputeGame.GameConstructorParams memory afterParams =
+            getGameConstructorParams(IFaultDisputeGame(address(OP_DGF.gameImpls(GameTypes.PERMISSIONED_CANNON))));
+        // Set the before params to match the after params, since that is the only thing that should have
+        // changed, the two sets of params should now have the same hash.
```

Comment:
typo
```suggestion
        // Set the before prestate to match the after prestate, since that is the only thing that should have
        // changed, the two sets of params should now have the same hash.
```